src/ResourceManager/AzureBatch/Commands.Batch/Models/NewPoolParameters.cs

```diff
@@ -65,7 +65,12 @@ public NewPoolParameters(BatchAccountContext context, string poolId, IEnumerable
         /// <summary>
         /// The target number of compute nodes to allocate to the pool.
         /// </summary>
-        public int? TargetDedicated { get; set; }
+        public int? TargetDedicatedComputeNodes { get; set; }
```

Comment:
same here

Reply:
My intent is to just leave this as is (no obsolete) since this is a breaking change, unless otherwise suggested by one of the powershell maintainers.

Reply:
This is actually only a type used internally in the cmdlets, it's not part of a cmdlet or return type. It's just used to store information about what request is being performed so we can pass it off to the C# SDK, so I don't think there's anything to really say in upcoming-breaking-changes.